src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,47 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("completeOrder method was called for userId: {}", userId);
```

Comment:
The log message in the 'completeOrder' method is not informative enough. According to the checklist, log messages should be informative. Consider including more details about the action being performed, for example, you can include the fact that the method is starting the process of completing an order.